dask/distributed/worker.py

```diff
@@ -65,6 +65,11 @@ class Worker(object):
         Port on which to listen to worker connections
     bind_to_workers: string
         Addresses from which we accept worker connections, defaults to *
+    heartbeat_on: bool
+        On/off switch for heartbeats, defaults to True
+    heartbeat_pulse: int
+        How frequently to send heartbeat messages to the scheduler in seconds,
+        defaults to 5
```

Comment:
Suggest single kwarg, `heartbeat` which is either an integer or `None`, for no heartbeats.  This might help to stem growing kwarg hell.